Image editor python script/Image_editing_script.py

```diff
@@ -0,0 +1,608 @@
+"""
+IMAGE EDITOR USING PILLOW LIBRARY.
+"""
+
+# Importing the image module from pillow library.
+
+# Imorting ImageFilter, ImageEnhance modules from pillow library.
+
+from PIL import Image, ImageFilter, ImageEnhance
+
+# Importing system module.
+
+import sys
+
+'''
+============================================================================
+^^^~~~~~~~~~~~~~~~~~~~~~|||||DEFINED FUNCTIONS|||||~~~~~~~~~~~~~~~~~~~~~~^^^
+============================================================================
+'''
+
+'''
+============== Function to show available options on CLI ==================
+'''
+
+
+def home():
+
+    print(" --> 1. Flip the image.")
+
+    print(" --> 2. Rotate the image by 90 degrees.")
+
+    print(" --> 3. Rotate the image by 180 degrees")
+
+    print(" --> 4. Blur the image")
+
+    print(" --> 5. Embossed image")
+
+    print(" --> 6. Apply more filters")
+
+    print(" --> 7. Contrast adjustments")
+
+    print(" --> 8. Increase sharpness")
+
+    print(" --> 9. quit")
+
+    print(" ")
+
+
+'''
+======================== Function to save edited image ========================
+'''
+
+
+# This function will ask the user if he wants the edited file to be saved
+
+def saving_edited_image():
+
+    print("========Do you want to save the edited image ?==========")
+
+    saving_choice = input("Type 1 for YES and 2 for NO : ")
+
+    if saving_choice in ("1", "2"):
+
+        if saving_choice == '1':
+
+            path_specified = input(r"===== Enter the path where you"
+                                   r" want to store the edited image."
+                                   r"also mention the name of image "
+                                   r"with extension=====")
+
+            # saving the image at given path
+
+            im.save(path_specified)
+
+            # final message after saving the image
+
+            print("====Image changes done and saved !! Keep editing====")
+
+        if saving_choice == '2':
+
+            # back to the main menu
+
+            print("~~~Back to the main menu~~~")
+
+            main()
+
+    else:
+
+        '''
+        if the user inputs value other than the specified ones,
+        this function will be called again.
+        '''
+
+        print("=====!!! PLEASE CHOOSE FROM SPECIFIED OPTIONS !!!=====")
+
+        saving_edited_image()
+
+
+'''
+================== Function for edited image preview =======================
+'''
+
+
+'''
+this function will ask the user if he wants a
+preview of edited image or not and will provide the same
+'''
+
+
+def edited_image_preview(image_name):
+
+    print("=======Do you want to preview the edited image ?=======")
+
+    preview_choice = input("Type 1 for yes and 2 for no : ")
```

Comment:
Be consistent with upper and lower case:
```suggestion
    preview_choice = input("Type 1 for YES and 2 for NO : ")
```